Library/Homebrew/software_spec.rb

```diff
@@ -344,3 +344,17 @@ def checksums
     checksums
   end
 end
+
+class PourBottleCheck
+  def initialize(formula)
+    @formula = formula
+  end
+
+  def reason(reason)
+    @formula.pour_bottle_check_unsatisfied_reason(reason)
```

Comment:
I think it's better to use `attr_accessor` rather than `attr_rw` here. The reason is `attr_rw` is majorly used as DSL method in the codebase. It's more straightful to use assignment here.

Also any chance to have a shorter variable name here?